start-site/src/main/resources/application.yml

```diff
@@ -1028,6 +1028,16 @@ initializr:
           description: Declarative REST Client. OpenFeign creates a dynamic implementation of an interface decorated with JAX-RS or Spring MVC annotations.
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-openfeign
+        - name: Spring Cloud LoadBalancer
+          id: cloud-loadbalancer
+          compatibilityRange: "[2.2.0.M4,2.2.x.BUILD-SNAPSHOT)"
+          description: Client-side load-balancing with Spring Cloud LoadBalancer.
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-starter-loadbalancer
+          links:
+            - rel: guide
```

Comment:
Can we please have a link to the documentation as well?

Reply:
It's not a separate project, so documentation for it is included in the SC Commons docs - there are different bits related to using it in different scenarios:
- https://cloud.spring.io/spring-cloud-static/spring-cloud-commons/2.2.0.M3/reference/html/#spring-resttemplate-as-a-load-balancer-client
- https://cloud.spring.io/spring-cloud-static/spring-cloud-commons/2.2.0.M3/reference/html/#spring-resttemplate-as-a-load-balancer-client
- https://cloud.spring.io/spring-cloud-static/spring-cloud-commons/2.2.0.M3/reference/html/#passing-your-own-load-balancer-client-configuration
Should I just add all of them?

Reply:
Can we rephrase the doc a bit to have a dedicated section for it please? If there is a dedicated starter, that's what I would expect.

Reply:
Have merged the changes: https://github.com/spring-cloud/spring-cloud-commons/pull/624 Once we do a new release (should be this week) and this gets published, will add it here.